RELEASE_NOTES.md

```diff
@@ -2,7 +2,194 @@
 
 ## 2.1.15 "Endora" - December 2, 2020
 
-* Linux system metrics monitor now ignores the following special mounts points by default:
+### Increase Scalyr Agent upload throughput, using multiple Scalyr team accounts. (BETA)
+**Note, the use of multiple sessions to upload logs to Scalyr is currently in beta. Please be sure to test use of this
+ feature before deploying it widely in your production environment.**
+
+
+#### Multi-worker configuration
+
+#### Use multiple workers.
+
+In a default configuration, the Scalyr Agent creates a single session with the Scalyr servers to upload all collected logs.
+This typically performs well enough for most users, but at some point, the log generation rate can exceed
+a single session's capacity (typically 2 to 8 MB/s depending on line size and other factors).
+You may increase overall throughput by increasing the number of sessions using the ``default_workers_per_pi_key`` option
+in the agent configuration. By default, the option value is ``1``.
+
+The main idea of the multi-worker configuration is to spread the produced load among multiple
+upload sessions(workers).
+
+For example:
+
+```
+{
+    "default_workers_per_api_key": 3,
+    "api_key": "<you_key>",
+    "logs": [
+      {
+        "path": "/var/log/app/*.log",
+      },
+    ]
+}
+```
+
+Here all the matched log files will be distributed among three workers. Each of them will establish their own independent
+upload session and will upload only a subset of the log files, thereby reducing the load on a particular worker.
+
+#### Use multiprocess workers.
+
+Even if the workers create separate sessions, by default, they will still run within the same Python process,
+limiting their resources to a single CPU core.
+This may become a problem when the number of log files is too big to be handled by a single CPU core, especially if
+additional features (such as [sampling](https://app.scalyr.com/help/scalyr-agent?teamToken=7bn1PMhjS%2F8335sB6vgDcQ--#filter)
+and [redaction](/https://app.scalyr.com/help/scalyr-agent?teamToken=7bn1PMhjS%2F8335sB6vgDcQ--#filter) rules)
+are enabled.
+
+This limitation can be addressed using the ``use_multiprocess_copying_workers`` option in the agent configuration.
+By default, the option value is ``false``
+
+```
+{
+    "use_multiprocess_copying_workers": true,
+    "default_workers_per_api_key": 3,
+    "api_key": "<you_key>",
+    "logs": [
+      {
+        "path": "/var/log/app/*.log",
+      },
+    ]
+
+}
+```
+
+Now each worker starts its own process and does not share the same CPU with other workers.
```

Comment:
If you are using Kubernetes, you will find it easier to configure these settings using their corresponding environment variables in the configmap for the Scalyr Agent DaemonSet:

    SCALYR_USE_MULTIPROCESS_COPYING_WORKERS=true
    SCALYR_DEFAULT_WORKERS_PER_API_KEY=3